flymake-shellcheck.el

```diff
@@ -117,10 +117,13 @@ Check for problems, then call REPORT-FN with results."
         (process-send-region flymake-shellcheck--proc (point-min) (point-max))
         (process-send-eof flymake-shellcheck--proc)))))
 
+(defvar sh-shell)
```

Comment:
Why do we need this declaration here?

Reply:
IIRC, flymake complains about `sh-shell` being a free variable